components/molecules/m-product-additional-info.vue

```diff
@@ -55,16 +54,8 @@ export default {
       return this.reviews.length
     }
   },
-  methods: {
-    onTabChange (index) {
-      let counter = 0
-      this.$children[0].$children.forEach(tab => {
-        counter++
-        if (tab._uid === index) {
-          this.$store.commit('ui/setActiveProductTab', counter);
-        }
-      })
-    }
+  destroyed () {
+    this.$store.commit('ui/setActiveProductTab', 1);
```

Comment:
there is no `this` with component scope in destroyed hook, please move it to beforeDestroy

Reply:
Done, thanks.